setup.md

```diff
@@ -39,53 +39,48 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
 
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
-- [Git](https://git-scm.com/downloads)
-- [Go](https://go.dev/doc/install) (v1.20+)
-- [GoReleaser](https://goreleaser.com/)
-- [Make](https://www.gnu.org/software/make/)
 - [jq](https://stedolan.github.io/jq/download/)
 
-## Step 1: Clone GUAC
+## Step 1: Download GUAC
 
-1. Clone GUAC to a local directory:
+1. Download the GUAC CLI `guacone` binary for your machine's OS and architecture
+   from the
+   [latest GUAC release](https://github.com/guacsec/guac/releases/latest). For
+   example Linux x86_64 is
+   [`guacone-linux-amd64'](https://github.com/guacsec/guac/releases/latest/download/guacone-linux-amd64).
 
-   ```bash
-   git clone https://github.com/guacsec/guac.git
-   ```
+1. Rename the binary to `guacone`, mark it executable if necessary, and add it
+   to your shell's path.
 
-2. Optional: If you want test data to use, clone GUAC’s test data:
+1. Download the compose files from the
+   [latest GUAC release](https://github.com/guacsec/guac/releases/latest). Look
+   for an attached file `guac-compose-<release-tag>.tar.gz`. At the time of
+   writing, this is:
+   [`guac-compose-v0.1.1.tar.gz`](https://github.com/guacsec/guac/releases/download/v0.1.1/guac-compose-v0.1.1.tar.gz).
 
-   ```bash
-   git clone https://github.com/guacsec/guac-data.git
-   ```
-
-3. Go to your GUAC directory (the rest of the steps need to be done from this
-   directory):
+1. Untar the compose files and change to that directory. (the rest of the steps
+   need to be done from this directory):
 
    ```bash
-   cd guac
+   tar zxvf guac-compose-v0.1.1.tar.gz
+   cd guac-compose
    ```
 
-## Step 2: Build the containers
+1. Optional: If you want test data to use,
+   [download and unzip GUAC’s test data.](https://github.com/guacsec/guac-data/archive/refs/heads/main.zip)
 
-From your GUAC directory, run:
-
-```bash
-make container
-```
-
-## Step 3: Start the GUAC server
+## Step 2: Start the GUAC server
 
 1. In another terminal, from your GUAC directory, run:
```

Comment:
Should this not be the same terminal window and within `guac-compose` directory?

Reply:
I think other terminal is fine, updated directory.